open-craft/opencraft
# Changes to support truncate for database name

Ticket: [FAL-2543](https://tasks.opencraft.com/browse/FAL-2543)

This PR is in support to the comment: https://github.com/open-craft/opencraft/pull/855#pullrequestreview-839088396 and acts as a part 1 for the PR: https://github.com/open-craft/opencraft/pull/855

Testing Instructions

1. Run the migration
2. Check database_name field is populated for all the instances
